src/backend/main.ts

```diff
@@ -760,6 +760,19 @@ ipcMain.on('createNewWindow', (e, url) => {
   new BrowserWindow({ height: 700, width: 1200 }).loadURL(url)
 })
 
+ipcMain.handle('isGameAvailable', async (e, args) => {
+  const { appName, runner } = args
+  const info = getGame(appName, runner).getGameInfo()
+  if (info && info.is_installed) {
+    if (existsSync(info.install.install_path!)) {
```

Comment:
```suggestion
    if (info.install.install_path && existsSync(info.install.install_path)) {
```

We currently aren't constraining `install_path`'s type based on `is_installed`, so it *could* be undefined even if `is_installed` is true. In the real world this is rather unlikely to happen, but point is that it *can* happen

I plan on revising our core types (`AppSettings`, `GameInfo`, and so on) soon:tm: so this can then no longer happen, but for now this would be safer

Reply:
good catch.